townsfolk/investigative/Juror

```diff
@@ -0,0 +1,9 @@
+**Juror** | Townsfolk Investigative/Group
```

Comment:
Hmm, I think Townsfolk Group would probably make more sense here (it can't be both).
I know we usually do TK > TI > TP > TM, but I'm not sure if we have a consistent place for TG. I'd say we should probably do TK > TG > TI >TP > TM which would be consistent with what we have right now and would mean that this role is a TG role.

Reply:
What if we did it like cult
As long as like a head juror lives then they can investigate
If they die/don't exist it's just a TG but doesn't disband
And juror is just a TG

Reply:
That's a decent solution but I didn't want to make it too similar too cult. It would be more unique if its vulnerability lies in the numbers and not a singular role.

Reply:
I agree. I think the CL thing should be unique to Cult.